pkg/scheduler/core/extender_test.go

```diff
@@ -112,20 +113,80 @@ type FakeExtender struct {
 	filteredNodes    []*v1.Node
 }
 
+// selectVictimsOnNodeByExtender checks the given nodes->pods map with predicates on extender's side.
+// Returns:
+// 1. More victim pods (if any) amended by preemption phase of extender.
+// 2. Number of violating victim (used to calculate PDB).
+// 3. Fits or not after preemption phase on extender's side.
+func (f *FakeExtender) selectVictimsOnNodeByExtender(
+	pod *v1.Pod,
+	node *v1.Node,
+	originVictims *schedulerapi.Victims,
+	pdbs []*policy.PodDisruptionBudget,
+) ([]*v1.Pod, int, bool) {
+	if fits, _ := f.runPredicate(pod, node); !fits {
```

Comment:
We should remove originalVictims from the node and then check the new pod. Understand the current tests here dont account for pods.